src/applications/facility-locator/api/MockLocatorApi.js

```diff
@@ -95,6 +99,19 @@ class MockLocatorApi {
             }
           } else if (locationType === pharmacyType) {
             locationsData = [providersDataJson.data[11]];
+          } else if (locationType === emergencyCareType) {
+            if (
+              !serviceType ||
+              serviceType === Object.keys(emergencyCareServices)[0]
+            ) {
+              locationsData = emergencyCareData.data;
+            }
+            if (serviceType === Object.keys(emergencyCareServices)[1]) {
+              locationsData = emergencyCareData.data.slice(0, 3);
+            }
+            if (serviceType === Object.keys(emergencyCareServices)[2]) {
+              locationsData = [emergencyCareData.data[4]];
+            }
```

Comment:
Can this block be moved into a separate function? LIke the linter warning says, this function is getting too long/complex.